composer/distributed/dist_strategy.py

```diff
@@ -171,9 +171,14 @@ def _recreate_fsdp_param_groups_from_unwrapped_opt_info(
     for fsdp_name, param in fsdp_wrapped_named_params:
 
         unwrapped_name = clean_tensor_name(fsdp_name)
-        # need to have a 1:1 mapping between a fsdp param name and the non-wrapped vanilla param name
-        retrieved_group_num = non_wrapped_param_names_to_group_num[unwrapped_name]
-        group_num_to_optimizer_info[retrieved_group_num]['params'].append(param)
+
+        # since we are iterating over all model.named_parameters() after fsdp wrapping, we need to check
```

Comment:
```suggestion
        # Since we are iterating over all model.named_parameters() after fsdp wrapping, we need to check
```
capitalize start of comment